build.gradle

```diff
@@ -7,7 +7,9 @@ plugins {
     id 'com.github.sherter.google-java-format' version '0.3.2'
     id "com.github.johnrengelman.shadow" version "2.0.1"
     id "net.ltgt.errorprone" version "0.0.10"
-    id 'ru.vyarus.animalsniffer' version '1.3.0'
+    id 'ru.vyarus.animalsniffer' version '1.4.2'
+    // './gradlew dependencyUpdates' will give you the updates of dependencies
```

Comment:
`./gradlew tasks` should list this documentation, I don't see the utility of this comment

Reply:
will remove